### PR TITLE
Feature show widget/layer in similar datasets section

### PR DESCRIPTION
**Pull request by pablopareja**
The following extra changes are included:

- Placeholder component
- Load datasets with no default widget/layer in both Explore and Explore Detail pages
- Hide Similar datasets section in Explore Detail when none was found

### CODE REVIEW

**Comment by pablopareja** [comment]
@mbarrenechea can you have a look at this? 
I have added the sidebar with the map view

File: src/components/pages/ExploreDetail.jsx [hunk 1]
```diff
@@ -64,6 +77,8 @@ class ExploreDetail extends React.Component {
   }
 
   getOpenMapButton(hasLayer) {
+    const { mapSectionOpened } = this.state;
+    const buttonText = (mapSectionOpened) ? 'Active' : 'Open in data map';
     const buttonClass = (hasLayer) ? '-active' : '';
```
Comment (mbarrenechea) [inline src/components/pages/ExploreDetail.jsx]: Let's use classnames instead of using ternary conditions

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Apart from this little thing, really good work

**Comment by pablopareja** [comment]
I've just fixed the classnames issue for the button. Besides that the default layer is already being displayed at the Explore Detail page.
@mbarrenechea can you check it out?

File: src/components/pages/ExploreDetail.jsx [hunk 2]
```diff
@@ -244,17 +249,12 @@ class ExploreDetail extends React.Component {
           <Map
             LayerManager={LayerManager}
             mapConfig={mapConfig}
-            layersActive={this.state.layersActive}
-            toggledDataset={this.props.toggledDataset}
+            layersActive={[dataset.detail.attributes.layer[0].attributes]}
```
Comment (mbarrenechea) [inline src/components/pages/ExploreDetail.jsx]: Why are you using the layer[0]? Shouldn't be the default one?

File: src/components/pages/ExploreDetail.jsx [hunk 3]
```diff
@@ -244,17 +249,12 @@ class ExploreDetail extends React.Component {
...
           />
-
-          {this.state.layersActive && this.state.layersActive.length &&
           <Legend
-            layersActive={this.state.layersActive}
+            layersActive={[dataset.detail.attributes.layer[0].attributes]}
```
Comment (mbarrenechea) [inline src/components/pages/ExploreDetail.jsx]: The same here

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: src/components/pages/ExploreDetail.jsx (hunk 2 above)
Comment (pablopareja) [reply src/components/pages/ExploreDetail.jsx]: fixed!

File: src/components/pages/ExploreDetail.jsx (hunk 3 above)
Comment (pablopareja) [reply src/components/pages/ExploreDetail.jsx]: fixed!

**Review by mbarrenechea — APPROVED** [review]

**Pull request merged**